torch/testing/_internal/common_methods_invocations.py

```diff
@@ -863,6 +863,25 @@ def reference_sigmoid(x):
                                     dtypes=[torch.bfloat16]),),
                        assert_autodiffed=True,
                        promotes_integers_to_float=True),
+        UnaryUfuncInfo('erfinv',
+                       ref=scipy.special.erfinv,
+                       decorators=(precisionOverride({torch.float16: 1e-2,
+                                                      torch.bfloat16: 1e-2,
+                                                      torch.float32: 1e-4}),),
+                       dtypes=all_types_and(torch.bool),
+                       dtypesIfCPU=all_types_and(torch.bool, torch.bfloat16),
+                       dtypesIfCUDA=all_types_and(torch.bool, torch.half),
+                       promotes_integers_to_float=True,
+                       handles_extremals=False,
```

Comment:
Interesting. Would you file an issue for erfinv not handling extremal values properly and cite it here?

Reply:
Turns out that it is fine if we set both `handles_extremals` and `handles_large_float` to true. Had set them to `False` while debugging for the SciPy discrepancy.
Have updated them again!

Thanks!